src/scripts/constants.ts

```diff
@@ -255,14 +264,18 @@ export module Constants {
 		export var cachedNotebooks = "notebooks";
 		export var currentSelectedSection = "curSection";
 		export var displayLanguageOverride = "displayLocaleOverride";
+		export var doNotPromptRatings = "doNotPromptRatings";
 		export var flightingInfo = "flightingInfo";
+		export var lastBadRatingDate = "lastBadRatingDate";
+		export var lastBadRatingVersion = "lastBadRatingVersion";
 		export var lastClippedDate = "lastClippedDate";
 		export var lastSeenVersion = "lastSeenVersion";
 		export var lastInvokedDate = "lastInvokedDate";
 		export var lastSeenTooltipTimeBase = "lastSeenTooltipTime";
 		export var lastClippedTooltipTimeBase = "lastClippedTooltipTime";
 		export var locale = "locale";
 		export var locStrings = "locStrings";
+		export var numClipSuccess = "numClipSuccess";
```

Comment:
I think that was the original value I had but then decided to value brevity over descriptiveness. Guess this could go back #Pending